ansible/roles_ocp_workloads/ocp4_workload_xray_data_pipeline/templates/storage/noobaa_system.yml.j2

```diff
@@ -0,0 +1,14 @@
+apiVersion: noobaa.io/v1alpha1
```

Comment:
Doesn't OCS automatically create this?

Reply:
Will check now that the deployment is OCS 4.5.
In the past you had to do this manually when deploying from YAMLs.
Anyway MCG is not deployed in this Lab if OCS respects the template. If the tests prove it's ok I can remove those lines.